outputs.tf

```diff
@@ -11,46 +11,46 @@ output "bucket_queue_name" {
   description = "Pubsub queue created for google bucket file upload events."
 }
 output "cluster_ca_certificate" {
-  value       = module.app_gke.cluster_ca_certificate
+  value       = var.create_gke ? module.app_gke.0.cluster_ca_certificate : null
```

Comment:
Let's remove this conditional. The purpose of this repository is to setup kubernetes, and I don't foresee anyone capable of setting up kubernetes on their own needing our tf to setup anything else. We must assume we're always going to setup kubernetes.

Reply:
@gls4  -- please check this link https://github.com/orgs/wandb/projects/17/views/1?pane=issue&itemId=60737094
It is a requirement that we should be able to install charts on the existing gke clusters(managed by some other terraform ) right ?

Reply:
@venky-wandb Can you explain why we're considering using Terraform to deploy into existing clusters? I'm hesitant about potential complexity. Do we have strong customer demand for this?

Reply:
@amanpruthi -- The purpose of this repo is to setup kubernetes and other resources. If kubernetes is already setup, this repo should not be used.